src/go/rpk/pkg/cli/cmd/generate/grafana_test.go

```diff
@@ -30,7 +30,7 @@ func TestGrafanaHostNoServer(t *testing.T) {
 	})
 	err := cmd.Execute()
 	require.Error(t, err)
-	require.Contains(t, err.Error(), "Get http://localhost:8888/metrics: dial tcp")
+	require.Contains(t, err.Error(), `Get "http://localhost:8888/metrics": dial tcp`)
```

Comment:
looks like this change broke the build. Can you ensure this commit passes the build.

Reply:
I you all don't mind, I'm going to try bumping Go to 1.15 again this time since I have more time to fix.

Reply:
sure, though bumping from go  13->15 should be a separate PR. unrelated to fixing issues on OSX

Reply:
I think this output changes depending on the OS.